frontend/src/App.jsx

```diff
@@ -1,21 +1,30 @@
-import React from "react";
+import React, { useEffect } from "react";
 import { BrowserRouter, Switch } from "react-router-dom";
 import { Route } from "react-router";
-import SuggestionsProvider from "./common/suggestion-provider";
+import { SuggestionsProvider } from "./common/context/suggestion-context";
 
 import Home from "./use-cases/home/home";
 import NotFound from "./use-cases/notfound";
 import Suggestion from "./use-cases/suggestion";
 import Callback from "./use-cases/callback";
 import "./App.css";
 
+const API = () => {
+    useEffect(() => {
+        window.location = "http://localhost:5000/api/login";
```

Comment:
why are you redirecting to localhost:5000?  This doesn't work in production right?

Reply:
When logging in, the frontend redirects the user to the backend at endpoint /api/login (which then redirects the user to Gamma). Since the frontend and the backend are on different hosts during development, redirecting to /api/login will redirect to localhost:3001 (frontend) and not localhost:5000 (backend). This component will redirect the user to the backend if ther user is redirected to /api in the frontend. In production both the backend and the frontend will use the same host, so redirecting to /api/login will no longer touch this component. The API component is only used during development.